snapcraft.yaml

```diff
@@ -222,31 +222,8 @@ parts:
       - valac
       - libgraphviz-dev
 
-  atk:
-    after: [ meson-deps ]
-    source: https://gitlab.gnome.org/GNOME/atk.git
-    source-tag: '2.38.0'
-    source-depth: 1
-    plugin: meson
-    meson-parameters:
-      - --prefix=/usr
-      - -Doptimization=3
-      - -Ddebug=true
-      - -Dintrospection=true
-      - -Ddocs=false
-    build-environment: *buildenv
-    override-build: |
-      set -eux
-      craftctl default
-      PC=$CRAFT_PART_INSTALL/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR/pkgconfig/atk.pc
-      sed -i 's#exec_prefix=/usr#exec_prefix=${prefix}#' $PC
-      sed -i 's#libdir=/usr/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR#libdir=${prefix}/lib/$CRAFT_ARCH_TRIPLET_BUILD_FOR#' $PC
-      sed -i 's#includedir=/usr/include#includedir=${prefix}/include#' $PC
-    build-packages:
-      - libgirepository1.0-dev
-
   at-spi2-core:
-    after: [ atk, meson-deps ]
+    after: [ meson-deps ]
```

Comment:
Have you reworked the build dependencies, so now really use only what really needs, instead of building everything "linearly"? I ask this because in the old .yaml for 42, this depended on gee...

Reply:
I don't understand why it was depending on gee, the Build-Depends of the deb are

```
Build-Depends: dpkg-dev (>= 1.22.5), debhelper-compat (= 13), dbus, libdbus-1-dev, dbus-broker [linux-any], libsystemd-dev [linux-any], libglib2.0-dev (>= 2.62), libxml2-dev, libx11-dev, libxtst-dev, meson (>= 0.63.0), pkgconf, gtk-doc-tools, gi-docgen, python3-sphinx, gir1.2-dbus-1.0-dev, gir1.2-glib-2.0-dev, gir1.2-gobject-2.0-dev, gobject-introspection | dh-sequence-gir, xauth <!nocheck>, xvfb <!nocheck>, at-spi2-core <!nocheck>
```

it seems to be working like that but I'm happy to change and add a depends if there is a reason for it

Reply:
Mmm... in that case, you should at least add glib and gobject-introspection in the "after" statement...

It was depending on gee because in the old file, each part depended on the previous one, and they were ordered in a way that ensured that, in that order, they would compile fine, although with the problem that modifying one part forced rebuilding all the later parts, no matter if they really depended on it or not.

Reply:
I guess adding after glib, g-i makes sense, I did that change and force pushed